src/nickel.rs

```diff
@@ -33,7 +38,39 @@ impl Nickel {
         // they don't like the default behaviour.
         middleware_stack.add_error_handler(DefaultErrorHandler);
 
-        Nickel { middleware_stack: middleware_stack }
+        Nickel { middleware_stack: middleware_stack, options: Nickel::default_options() }
+    }
+    
+    /// Gets the default options for the framework
+    ///
+    /// # Examples
+    /// ```{rust}
+    /// # #[macro_use] extern crate nickel;
+    /// # fn main() {
+    /// use nickel::{Nickel, Options};
+    /// let mut server = Nickel::new();
+    /// let options = Options{output_on_listen: false, .. Nickel::default_options()};
+    /// server.set_options (options);
+    /// # }
+    /// ```
+    pub fn default_options() -> Options {
+        Options {output_on_listen: true}
+    }
+    
+    /// Sets the options for this instance
+    ///
+    /// # Examples
+    /// ```{rust}
+    /// # #[macro_use] extern crate nickel;
+    /// # fn main() {
+    /// use nickel::{Nickel, Options};
+    /// let mut server = Nickel::new();
+    /// let options = Options{output_on_listen: false, .. Nickel::default_options()};
+    /// server.set_options (options);
+    /// # }
+    /// ```
+    pub fn set_options(&mut self, options: Options) {
```

Comment:
I think this should be `options_mut` as that would allow `get` or `set` behaviour?